### ISSUE TITLE
stress.fathom causes stack overflow on Windows

### ISSUE THREAD

**Issue by wezm** [yeslogic/fathom#325]

**Comment by brendanzab** [comment]
Yeah, I was actually running into this when attempting to upgrade `scoped-arena` to version `0.4.1`. I'm not sure this test (along with `succeed/prelude.fathom` and `fail/paradoxes/hurkens.fathom`) will survive #316 though. These were more there as I was doing initial tests of the elaborator.

**Comment by wezm** [comment]
Noticed a couple of ways to handle this in a URLO forum thread:

1. [Spawn a thread to do the work with a custom stack size](https://users.rust-lang.org/t/what-can-i-do-to-avoid-thread-main-has-overflowed-its-stack-when-working-with-large-arrays/77091/3?u=wezm)
1. [Increase the stack size of the main thread at compile time with a build script](https://users.rust-lang.org/t/what-can-i-do-to-avoid-thread-main-has-overflowed-its-stack-when-working-with-large-arrays/77091/8?u=wezm)
